File: app/src/main/java/io/defy/chicken/lover/network/response/ChickenSelectHistoryRes.java
package io.defy.chicken.lover.network.response;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class ChickenSelectHistoryRes {
    @SerializedName("result")
    String result;

    @SerializedName("result_array")
    List<Obj> result_array;;

    public String getResult() {
        return result;
    }

    public List<Obj>  getResultArray() {
        return result_array;
    }

    public class Obj {
        public String get_id() {
            return _id;
        }

        public String getName() {
            return name;
        }

        public String getChicken_name() {
            return chicken_name;
        }

        public String getChicken_brand() {
            return chicken_brand;
        }

        public String getSelect_date() {
            return select_date;
        }

        @SerializedName("_id")
        String _id;
        @SerializedName("name")
        String name;
        @SerializedName("chicken_name")
        String chicken_name;
        @SerializedName("chicken_brand")
        String chicken_brand;
        @SerializedName("select_date")
        String select_date;

    }
}

File: app/src/main/java/io/defy/chicken/lover/network/response/ChickenSelectHistoryRes.kt
package io.defy.chicken.lover.network.response

import com.google.gson.annotations.SerializedName

data class ChickenSelectHistoryRes(
    @SerializedName("result") var result: String? = null,
    @SerializedName("result_array") var result_array: List<InnerArray> = ArrayList()
) {
    data class InnerArray(
        @SerializedName("_id") var _id : Int = 0,
        @SerializedName("name") var name: String,
        @SerializedName("chicken_name") var chicken_name: String,
        @SerializedName("chicken_brand") val chicken_brand : String,
        @SerializedName("select_date") var select_date: String
    )
}